lib/debuglet.js

```diff
@@ -215,7 +217,18 @@ Debuglet.prototype.scheduleBreakpointFetch_ = function(seconds) {
 
         default:
           that.logger_.info('\t' + response.statusCode + ' completed.');
-          that.updateActiveBreakpoints_(body.breakpoints || []);
+          var bps = (body.breakpoints || []).filter(function(bp) {
+            var action = bp.action || 'CAPTURE';
+            if (action !== 'CAPTURE') {
+              that.logger_.warn('Found breakpoint with invalid action:', action);
```

Comment:
You could get more of this code pulled into rejectBreakpoint. Sorry I wasn't clearer earlier. I think there should be minimal logic here. Check action; call reject and return false.